chapters/glossary.tex

```diff
@@ -47,8 +47,11 @@ \chapter{Glossary}\doublelabel{glossary}
 classes. (See \autoref{inheritance-extends-clause}.)
 
 \textbf{binding equation}: Either a declaration equation or an element
-modification for the value of the variable. A component with a binding
+modification for the value of the variable. In a non-function a component with a binding
 equation has its value bound to some expression. (See \autoref{equation-categories}.)
+In a function an input component uses the value of the binding equation as default value,
+a non-input component starts with that value.
```

Comment:
Does this also consider binding equations for outputs, e.g., `function 'the answer' output Real y=42; end 'the answer';` or should outputs additionally be mentioned (since start value might be confusing)?

Reply:
I would say so, since outputs and protected variables in functions are treated the same; they start with that value (before executing the body of the function) and can then be changed in the body of the function.

Reply:
OK. Fine for me.